janusgraph-codepipelines-ci/pom.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.janusgraph</groupId>
+        <artifactId>janusgraph</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>janusgraph-codepipelines-ci</artifactId>
+    <name>JanusGraph-CodePipeline CI: Distributed Graph Database</name>
+    <url>http://janusgraph.org</url>
+    <properties>
+        <top.level.basedir>${basedir}/..</top.level.basedir>
+        <aws.sdk.version>1.11.86</aws.sdk.version>
```

Comment:
I want to limit exposure to AWS specific APIs to the specific places where it is used / can be used (AWS ES, HBase on EMR, and now, CodePipelines). Currently the only code dependency is in janusgraph-codepipelines-ci package, so lets keep the SDK version here for now. If we find we need the SDK in more than one place, then I think that is the time to move it up into the parent POM.